Doppler.Sap.Test/Doppler.Sap.Test.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="Moq" Version="4.14.5" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
```

Comment:
This package reference is not used, right?

Reply:
You are right, but that reference is included in the basic template, maybe we already have a nice, free, and VS independent test coverture report out of the box?

Reply:
There is more information here: https://github.com/coverlet-coverage/coverlet

I did not read it yet.